django_fido/settings.py

```diff
@@ -32,7 +32,8 @@ class DjangoFidoSettings(AppSettings):
         access_token=StringSetting(required=True),
         url=StringSetting(default='https://mds2.fidoalliance.org/'),
         timeout=Setting(default=3, validators=[timeout_validator]),
-
+        certificate=FileSetting(),
+        crl_list=cast(list, NestedListSetting(inner_setting=FileSetting(), default=[])),
```

Comment:
* The cast shouldn't be necessary any more.
* Why don't we use `/etc/ssl/certificates/*` as default?

Reply:
> * Why don't we use `/etc/ssl/certificates/*` as default?
It wouldn't validate as the TOC is signed by a certtificate signed with FIDO Aliance root certificate, which would be missing in `/etc/ssl/certificates/*` (and there is probably zero reasons to put this specific certificate there...)

Reply:
Anyway, there probably should be a list of certificates to support rotation when new root certificate is published.

Reply:
This is already a list, so rotation should be possible in the current state.

Reply:
Ah, for the root certificate itself. Yes, that should be possible...